app/design/frontend/Magento/luma/Magento_Theme/layout/default_head_blocks.xml

```diff
@@ -8,10 +8,5 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
-        <font src="fonts/opensans/light/opensans-300.woff2"/>
-        <font src="fonts/opensans/regular/opensans-400.woff2"/>
-        <font src="fonts/opensans/semibold/opensans-600.woff2"/>
-        <font src="fonts/opensans/bold/opensans-700.woff2"/>
-        <font src="fonts/Luma-Icons.woff2"/>
```

Comment:
those included fonts were added here in order to have some font preload that google pagespeed is telling for. Are you sure it's causing  layout shift?

Reply:
@ihor-sviziev i'm not sure why this located here. But if you remove it icon still show exactly like before. We can safe remove this

Reply:
Previous this from blank theme. After some commit it's moved to luma. Default icon and font imported via less

Reply:
Yes, they are imported from less files, but it adds font preload, like this:
```html
<link rel="preload" as="font" crossorigin="anonymous" href="https://www.yourdomain.com/static/version1605168925/frontend/Magento/Luma/en_US/fonts/Luma-Icons.woff2" />
```

Reply:
I think this is unnecessary. It Take more request to browser

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content
So i think i need revert this change back to previous

Reply:
cc @krzksz what do you think

Reply:
Reverted this back to previously